platform/android/app/src/main/kotlin/coop/polypoly/polypod/PodApi.kt

```diff
@@ -106,7 +105,7 @@ open class PodApi(
 
     private suspend fun handlePolyOutWriteFile(args: List<Value>): Value {
         logger.debug("dispatch() -> polyOut.readFile")
-        val result = polyOut.writeFile(
+        polyOut.writeFile(
```

Comment:
This seems like it wants at least a TODO comment - sure, we're not using `result`,  but just assuming the operation worked seems dangerous.